misc/config_tools/data/sample_launch_scripts/apl-mrb/launch_user_vm.sh

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This script should be removed actually. Is there any reason we couldn't just delete it?

Reply:
In 2.7 we don't delete the script. We will delete for re-generate the script in later commit. This will be in KunhuiX's patch.